src/Microsoft.AspNet.Server.Kestrel/ServiceContext.cs

```diff
@@ -31,6 +33,8 @@ public ServiceContext(ServiceContext context)
 
         public IKestrelTrace Log { get; set; }
 
+        public IHttpContextFactory HttpContextFactory { get; set; }
```

Comment:
Yes we use object initializers. Like in https://github.com/aspnet/KestrelHttpServer/blob/dev/src/Microsoft.AspNet.Server.Kestrel/ServerFactory.cs#L58-L62